master/reference/calicoctl/resources/networkset.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Network Set Resource (GlobalNetworkSet)
+---
+
+A network set resource (GlobalNetworkSet) represents an arbitrary set of IP subnetworks/CIDRs, allowing it
+to be matched by Calico policy.  Network sets are useful for applying policy to traffic
+coming from (or going to) external, non-Calico, networks.
+
+The metadata for each network set includes a set of labels.  When Calico is calculating the set of
+IPs that should match a source/destination selector within a
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) rule, it includes
+the CIDRs from any network sets that match the selector.
+
+> **Important**: Since Calico matches packets based on their source/destination IP addresses, 
+> Calico rules may not behave as expected if there is NAT between the Calico-enabled node and the
+> networks listed in a network set.  For example, in Kubernetes, incoming traffic via a service IP is
+> typically SNATed by the kube-proxy before reaching the destination host so Calico's workload 
+> policy will see the kube-proxy's host's IP as the source instead of the real source.
+{: .alert .alert-danger}
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `globalnetworkset`, `globalnetworksets`.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkSet
+metadata:
+  name: a-name-for-the-set
+  labels:
+    role: external-database
+spec:
+  nets:
+  - 198.51.100.0/28
+  - 203.0.113.0/24
+```
+
+### GlobalNetworkSet Definition
+
+#### Metadata
+
+| Field       | Description                                | Accepted Values   | Schema  |
+|-------------|--------------------------------------------|-------------------|---------|
+| name        | The name of this network set.              | Lower-case alphanumeric with optional '-'  | string  |
```

Comment:
maybe we should make this similar to other resources: "Lower-case alphanumeric string with optional `.`,` _`, or `-`." , I realize other resources don't mention `Lower-case` but they should, but doesn't have to be part of this PR